File: src/main/java/com/back/pinco/domain/user/dto/UserResBody/LoginResponse.java
package com.back.pinco.domain.user.dto.UserResBody;

import com.back.pinco.domain.user.dto.UserDto;

public record LoginResponse(
        Long id,
        String email,
        String userName
) {
    public LoginResponse(UserDto userDto) {
        this(
                userDto.id(),
                userDto.email(),
                userDto.userName()
        );
    }
}

File: src/main/java/com/back/pinco/domain/user/dto/UserResBody/LoginResponse.kt
package com.back.pinco.domain.user.dto.UserResBody

import com.back.pinco.domain.user.dto.UserDto

data class LoginResponse(
    val id: Long?,
    val email: String,
    val userName: String
) {
    constructor(userDto: UserDto) : this(
        userDto.id,
        userDto.email,
        userDto.userName
    )
}
